src/components/Main.tsx

```diff
@@ -0,0 +1,687 @@
+        options={{
+          title: s.strings.title_settings
+        }}
+        listeners={{
+          focus: () => dispatch(showReEnableOtpModal())
+        }}
+      />
+      <Stack.Screen
+        name="transactionDetails"
+        component={ifLoggedIn(TransactionDetailsScene)}
+        options={{
+          headerTitle: () => <TransactionDetailsTitle />
+        }}
+        listeners={{
+          focus: async () => requestPermission('contacts')
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletImport"
+        component={ifLoggedIn(CreateWalletImportScene)}
+        options={{
+          headerRight: () => null
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletCompletion"
+        component={ifLoggedIn(CreateWalletCompletionScene)}
+        options={{
+          headerLeft: () => null,
+          headerRight: () => null
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletAccountSetup"
+        component={ifLoggedIn(CreateWalletAccountSetupScene)}
+        options={{
+          title: s.strings.create_wallet_create_account,
+          headerRight: () => <HeaderTextButton type="help" placement="right" />
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletAccountSelect"
+        component={ifLoggedIn(CreateWalletAccountSelectScene)}
+        options={{
+          title: s.strings.create_wallet_account_activate,
+          headerRight: () => <HeaderTextButton type="help" placement="right" />
+        }}
+      />
+
+      <Stack.Screen
+        name="transactionList"
+        component={ifLoggedIn(TransactionList)}
+        listeners={{
+          focus: async () => requestPermission('contacts')
+        }}
+        options={{
+          title: ' '
+        }}
+      />
+
+      <Stack.Screen name="stakeModify" component={ifLoggedIn(StakeModifyScene)} />
+      <Stack.Screen name="stakeOptions" component={ifLoggedIn(StakeOptionsScene)} />
+      <Stack.Screen name="stakeOverview" component={ifLoggedIn(StakeOverviewScene)} />
+      <Stack.Screen name="fioStakingOverview" component={ifLoggedIn(FioStakingOverviewScene)} />
+      <Stack.Screen name="fioStakingChange" component={ifLoggedIn(FioStakingChangeScene)} />
+
+      <Stack.Screen
+        name="manageTokens"
+        component={ifLoggedIn(ManageTokensScene)}
+        options={{
+          title: '',
+          headerRight: () => null
+        }}
+      />
+      <Stack.Screen
+        name="editToken"
+        component={ifLoggedIn(EditTokenScene)}
+        options={{
+          headerRight: () => null
+        }}
+      />
+      <Stack.Screen
+        name="transactionsExport"
+        component={ifLoggedIn(TransactionsExportScene)}
+        options={{
+          title: s.strings.title_export_transactions,
+          headerRight: () => null
+        }}
+      />
+      <Stack.Screen
+        name="pluginViewBuy"
+        // @ts-expect-error-error
+        component={ifLoggedIn(GuiPluginViewScene)}
```

Comment:
Go to GuiPluginViewScene, line 119 in `OwnProps`, and change the route prop to `route: RouteProp<'pluginView' | 'pluginViewBuy' | 'pluginViewSell'>`. Now your `// @ts-expect-error` goes away!

Reply:
Thanks for this catch!